### DISCUSSION TITLE
Improve side navigation

### DISCUSSION THREAD

**Discussion by moshetanzer** [coollabsio/coolify#3623] [Improvement Requests]
### Error Message and Logs

Hi @andrasbacsai 

If you don't mind the following suggestions:
- To give coolify a bit more of a cleaner look we should def remove `profile` and `notifications` from side navigation and put them into settings (they are not generally accessed frequently and people are used to then being in settings).

-I would go so far to suggest that even `destatnations`, `keys and tokens` and `sources` should also be in settings (they are not accessed frequently at all in most use cases).

So too onboarding should not be in side navigation - i haven't touched it since I setup coolify and have seen a few issues or comments about it.

thanks for your amazing work!

### Steps to Reproduce

1.
2.
3.
4.

### Coolify Version

latest

### Are you using Coolify Cloud?

No (self-hosted)

**Comment by peaklabs-dev** [comment]
Just to let you know, a UI overhaul is planned for the future.

**Comment by moshetanzer** [comment]
Well aware of that. Just opened as issue as this is low hanging fruit and easy to improve coolify ux. like onboarding... sources... it really seems like wanted space.
